src/components/DashboardHome/Dashboard.js

```diff
@@ -10,12 +10,12 @@ const Dashboard = () => {
     dispatch(creators.getCurrentUser())
   }, [dispatch])
   const user = useSelector(state => state.user.user)
-  const url = `${window.location.origin.toString()}/store/${user &&
+  const url = `${window.location.origin.toString()}/store/` + encodeURIComponent(`${user &&
```

Comment:
The whole essence of adding the `id` is for the backend and it shouldn't be removed

Reply:
Isn't this the public URL that the seller will give to buyers? If so, I think it shouldn't include the ID. Also, we already validate that the store name is unique, so why do we also need ID? Can't we find stores by their name?

Reply:
If the store names are unique, I think store name in url is fine. Does this change anything on the backend though?

Reply:
Yes, it'll need a corresponding change on the back end, but I don't think we've got to the point where we're integrated yet.

Reply:
So, I'd close this PR because 
1. To query the database with the store name won't be consistent based on some formating
2. The products are only linked with the `storeId` and not store name
3. To achieve this, we would have to spend extra time refactoring the code.
Thanks

Reply:
I think we should leave as ID for now. changing to store name isn't as straight forward as I initially thought. Since the change is mostly aesthetic, we can wait till we have a firm grasp of all ramifications and time to implement them before making this change.